src/components/AppBar/Drawer.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import Box from '@mui/material/Box';
+import Drawer from '@mui/material/Drawer';
+import List from '@mui/material/List';
+import Divider from '@mui/material/Divider';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import IconButton from '@mui/material/IconButton';
+import SailingIcon from '@mui/icons-material/Sailing';
+import Avatar from '@mui/material/Avatar';
+import { useFetchLogoutMutation } from '../../services/AuthServices';
+import Icon from '../../utils/Icon'
+import { MENU_ITEMS, RouterLinks } from '../../utils/consts'
+
+export default () => {
+  const [fetchLogout] = useFetchLogoutMutation();
+  const [isOpened, setIsOpened] = React.useState(false)
+  const navigate = useNavigate()
+
+  const toggleDrawer = (state: boolean): void => setIsOpened(state)
```

Comment:
❗️
Тогглы должны переключать состояние, а не принимать и сетить
```suggestion
  const toggleDrawer = (): void => setIsOpened(!isOpened)
```